src/common/snippets/src/op/store.cpp

```diff
@@ -12,13 +12,19 @@ namespace ngraph {
 namespace snippets {
 namespace op {
 
-snippets::op::Store::Store(const Output<Node>& x, const size_t count, const size_t offset) : MemoryAccess({x}, count, offset) {
+snippets::op::Store::Store(const Output<Node>& x, const size_t count, const size_t offset) : MemoryAccess({x}) {
     constructor_validate_and_infer_types();
+    set_output_port_descriptor({count, offset}, 0);
```

Comment:
`set_output_port_descriptor` is (almost?) always called with an rvalue, so why don't we add a corresponding parameter to a `MemoryAccess` constructor? The parameter could even be mandatory to protect us from the cases when one simply forgets to manually call `set_output_port_descriptor` (and this will happen sooner or later).